integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/util/ApictlUtils.java

```diff
@@ -133,7 +135,7 @@ public static void createProject(String openApiFile, String apiProjectName) thro
      * @throws CCTestException if apictl was unable to create the project
      */
     public static void createProject(String openApiFile, String apiProjectName, String backendCert, String deployEnvYamlFile,
-                                     String interceptorCert)
+                                     String interceptorCert, String apiYamlFile)
```

Comment:
Shall we do overloading on the method so that you won't have this many diff due to new variable introduction.

Reply:
fixed with: https://github.com/wso2/product-microgateway/pull/3142/commits/c77b3dced5fc5331d2b5157ba174af38261d2fae